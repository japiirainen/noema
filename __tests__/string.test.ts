import { string } from "../src/index.js"
import { expectParseFailure, expectParseSuccess } from "./helpers.js"

describe("string", () => {
  test("should parse strings", () => {
    expectParseSuccess(string, "")
  })

  test("should fail to parse non-strings", () => {
    expectParseFailure(string, 0)
  })
})
