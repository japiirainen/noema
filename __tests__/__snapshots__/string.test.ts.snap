// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`string should fail to parse non-strings 1`] = `
[
  {
    "code": "invalid_string",
    "message": "Expected value to be a string",
    "path": "",
    "value": 0,
  },
]
`;
